### PR TITLE
fix: l2-bridge-base moved  🚨🚨🚨 Base bridge balance mismatch 🚨🚨🚨from l…

### PR DESCRIPTION
**Pull request by sergeyWh1te**
Part of https://github.com/lidofinance/alerting-forta/issues/526

### CODE REVIEW

File: l2-bridge-base/src/app.ts [hunk 1]
```diff
@@ -65,14 +68,13 @@ export class App {
         transports: [new Winston.transports.Console()],
       })
 
-      const baseRpcURL = FortaGuardClient.getSecret()
-
       const baseNetworkID = 8453
-      const nodeClient = new ethers.providers.JsonRpcProvider(baseRpcURL, baseNetworkID)
+      const baseProvider = new ethers.providers.JsonRpcProvider('https://base.llamarpc.com', baseNetworkID)
```
Comment (TheDZhon) [inline l2-bridge-base/src/app.ts:72]: maybe [base.drpc.com](https://base.drpc.org)? :)

File: l2-bridge-base/src/utils/constants.ts [hunk 2]
```diff
@@ -9,6 +11,8 @@ export type L2_ERC20_TOKEN_GATEWAY_TYPE = OProxyContract
 export type BASE_WST_ETH_BRIDGED_TYPE = OProxyContract
 
 export type Address = {
+  WSTETH_ADDRESS: string
+  BASE_L1ERC20_TOKEN_BRIDGE: string
   GOV_BRIDGE_ADDRESS: string
   L2_ERC20_TOKEN_GATEWAY_ADDRESS: string
   BASE_WST_ETH_BRIDGED_ADDRESS: string
```
Comment (TheDZhon) [inline l2-bridge-base/src/utils/constants.ts:18]:
```suggestion
  L1_WSTETH_ADDRESS: string
  BASE_L1ERC20_TOKEN_BRIDGE: string
  GOV_BRIDGE_ADDRESS: string
  L2_ERC20_TOKEN_GATEWAY_ADDRESS: string
  L2_WSTETH_ADDRESS: string
```

**Review by TheDZhon — APPROVED** [review]
LGTM, only a few nitpicks

File: l2-bridge-base/src/app.ts (hunk 1 above)
Comment (sergeyWh1te) [reply l2-bridge-base/src/app.ts:72]: No. It lives with that provider. Also, tests covered app behaviuor

**Pull request merged**